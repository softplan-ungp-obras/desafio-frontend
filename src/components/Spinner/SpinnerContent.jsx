import React from 'react'
import { Spinner, SpinnerContentStyled } from './styled'

const SpinnerContent = () => (
  <SpinnerContentStyled>
    <Spinner />
  </SpinnerContentStyled>
)

export default SpinnerContent
