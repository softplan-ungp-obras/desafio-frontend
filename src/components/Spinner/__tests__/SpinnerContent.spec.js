import React from 'react'
import { shallow } from 'enzyme'
import SpinnerContent from '../SpinnerContent'

describe('SpinnerContent', () => {
  it('should render', () => {
    const spinner = shallow(<SpinnerContent />)

    expect(spinner).toMatchSnapshot()
  })
})
