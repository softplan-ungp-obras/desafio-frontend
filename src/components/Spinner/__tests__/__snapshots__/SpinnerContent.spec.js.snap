// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SpinnerContent should render 1`] = `
<SpinnerContentStyled>
  <Spinner />
</SpinnerContentStyled>
`;
